Gruntfile.js

```diff
@@ -355,6 +355,6 @@ module.exports = function(grunt) {
   grunt.registerTask('minify', ['bower','clean', 'build', 'minall']);
   grunt.registerTask('webserver', ['connect:devserver']);
   grunt.registerTask('package', ['bower','clean', 'buildall', 'minall', 'collect-errors', 'docs', 'copy', 'write', 'compress']);
-  grunt.registerTask('ci-checks', ['ddescribe-iit', 'merge-conflict', 'jshint', 'jscs']);
+  grunt.registerTask('ci-checks', ['merge-conflict', 'jshint', 'jscs']);
```

Comment:
It was convenient having all these tests in a single task, so one can run them before pushing to master with a single command. I would prefer to leave `ci-checks` as is and create a new task excluding the task that we choose (be it `ddescribe-iit` or `jscs`).